src/main.py

```diff
@@ -41,20 +42,32 @@ def mensa(update, context):
 
 
 def andre(update, context):
-    context.bot.send_message(chat_id=update.message.chat_id, text="Höhöhö Reichenbach!")
+    context.bot.send_message(chat_id=update.message.chat_id, text="H�h�h� Reichenbach!")
+
+
+def leon(update, context):
+    joke = dadJoke()
+    context.bot.send_message(chat_id=update.message.chat_id, text=joke)
+    
 
+def loen(update, context):
+    joke = dadJoke()
+    translator = translate.Translator(from_lang = 'en', to_lang = 'de')
+    translatedJoke = translator.translate(joke)
+    context.bot.send_message(chat_id=update.message.chat_id, text=translatedJoke)
+    
 
-def dadJoke(update, context):
+def dadJoke():
     headers = {'Accept': 'text/plain '}
     resp = requests.get("https://icanhazdadjoke.com/", headers=headers)
     if not resp.ok:
-        context.bot.send_message(chat_id=update.message.chat_id, text="I failed miserably. Disgrace!")
-        return
-    context.bot.send_message(chat_id=update.message.chat_id, text=resp.text)
+        return "I failed miserably. Disgrace!"
+    return resp.text
 
 
 def maxime(update, context):
     context.bot.send_sticker(chat_id=update.message.chat_id, sticker="CAADBQADfAMAAukKyAPfAAFRgAuYdNoWBA")
+    
```

Comment:
This extra line is correct though, we can leave it.

Reply:
Why is it still marked as a change, i removed the extra line?